content/docs/user-guide/gems/repositories/overview.md

```diff
@@ -0,0 +1,191 @@
+
+### Using a Template
+
+1. Create a new gem repo using `o3de.bat`.  This example uses a localhost URL as the repository URI for testing.
+
+    ```
+    scripts\o3de.bat create-from-template -tp Templates\GemRepo -dp <repo path on disk> -r ${RepoURI} http://localhost:8080/
+    ```
+
+1. Open the `repo.json` file in a text editor and update all the fields except the gem's list - we'll update that when adding a gem to the repository in the next step.
+    ```
+    {
+        "repo_name":"RepositoryName",
+        "origin":"RemoteOriginName",
+        "repo_uri": "http://localhost:8080/",
+        "summary": "Example Repository",
+        "additional_info": "Example repository hosted on local http server.",
+        "last_updated": "10/26/2021 04:56pm",
+        "gems": [
+        ]
+    }
+    ```
+
+### Add a Gem to the Gem Repository
+1.  Make a folder inside your Gem Repository for the Gem.  In this example we'll assume the gem is named "RemoteExample" and make a folder named "RemoteExampleV1" and place a `gem.json` in there that looks like this:
+    ```
+    {
+        "gem_name": "RemoteExample",
+        "display_name": "Example Remote Gem",
+        "license": "Apache-2.0 Or MIT",
+        "origin":"RemoteOriginName",
+        "origin_uri": "http://localhost:8080/RemoteExampleV1/RemoteExampleV1.zip",
+        "sha256":"",
+        "type": "Code",
+        "summary": "An example remote gem.",
+        "canonical_tags": [
+            "Gem"
+        ],
+        "user_tags": [
+            "Gameplay"
+        ],
+        "icon_path": "preview.png",
+        "requirements": "",
+        "documentation_url": "https://o3de.org/docs",
+        "dependencies": []
+    }
+    ```
+
+    The `origin_uri` points to the location of the gem `.zip` archive that will be downloaded
+
+1. Create a Gem, or take an existing Gem, zip up the contents and place it in the "RemoteExampleV1" folder, and name it `RemoteExampleV1.zip`
+1. Generate the sha256 for your Gem .zip archive and add that value to the "sha256" field in the `gem.json` file that is outside the .zip.  On Windows, you can use the `certutil` program to generate the SHA256 value like this:
+    ```
+    certutil -hashfile C:/path/to/gem.zip SHA256
+    ```
+    {{< note >}}
+    It is possible to leave out the sha256 field for testing, but discouraged
+    {{< /note >}}
+1. Update your repo.json file and add the URL for the gem folder.  In this example we'll use the localhost URL for testing.
+    ```
+    {
+        ...
+        "gems": [
+            "http://localhost:8080/RemoteExampleV1/"
+        ]
+    }
+    ```
+
+### Testing your local Gem Repository
+
+To test your Gem Repository before you upload it to a server, you can host a local server using Python or register the folder path to your Gem Repository, but if you register the folder path, you'll need to modify the URIs to use absolute paths like `file://c/path/to/gem` instead of `http://localhost:8080/gem`
```

Comment:
```suggestion
To test your Gem repository before you upload it to a server, you can host a local server using Python or register the folder path to your Gem repository. If you register the folder path, you'll need to modify the URIs to use absolute paths like `file://c/path/to/gem` instead of `http://localhost:8080/gem`.
```